### ISSUE TITLE
Whitelist tab order

### ISSUE THREAD

**Issue by chrisd08** [chrisd08/Cryptov8#4]
On the 'Cryptov8 ICO Whitelist' Page, I think the tab order should be First Name, Last Name, Email, Expected ..... rather than as currently First Name, Email, ERC-20 address....

**Comment by chrisd08** [comment]
Changed to desired tab order
